jenkins_jobs/modules/helpers.py

```diff
@@ -510,7 +513,10 @@ def trigger_get_parameter_order(registry, plugin):
     return None
 
 
-def trigger_project(tconfigs, project_def, param_order=None):
+def trigger_project(tconfigs, project_def, registry, param_order=None):
+
+    info = registry.get_plugin_info("parameterized-trigger")
```

Comment:
You cannot do it here, it's a common code shared with another publisher called `pipeline`.